theories/distributions/Distr.ec

```diff
@@ -784,15 +784,21 @@ lemma dmap_ll (d : 'a distr) (f : 'a -> 'b) :
   is_lossless d => is_lossless (dmap d f).
 proof. by rewrite /is_lossless weight_dmap. qed.
 
-lemma dmap_uni (d : 'a distr) (f : 'a -> 'b) : 
-  injective f => is_uniform d => is_uniform (dmap d f).
-proof. 
+lemma dmap_unig (d : 'a distr) (f : 'a -> 'b) :
```

Comment:
`dmap_unig` -> `dmap_uni_in_inj`

Reply:
@strub How about `dmap_uni_support` instead in the spirit of `mu_eq_support`?